Source/WebKit/WPEPlatform/wpe/WPEClipboard.cpp

```diff
@@ -399,6 +414,26 @@ void wpe_clipboard_content_set_bytes(WPEClipboardContent* content, const char* f
     content->buffers->add(g_intern_string(format), bytes);
 }
 
+/**
+ * wpe_clipboard_content_get_bytes:
+ * @content: a #WPEClipboardContent
+ * @format: a format
```

Comment:
How is this format specified? Is it a MIME type? I think that should be mentioned in the documentation.